zgrab_schema.py

```diff
@@ -773,7 +773,7 @@
                 "client_to_server_languages":ListOf(String()),
                 "server_to_client_languages":ListOf(String()),
                 "first_kex_follows":Boolean(),
-                "reserved":Short(),
+                "reserved":Unsigned32BitInteger(),
```

Comment:
[RFC 4253](https://www.ietf.org/rfc/rfc4253.txt) says this is a reserved `uint32` (see page 16 of the RFC)/